content/scalers/apache-kafka-topic.md

```diff
@@ -132,4 +132,4 @@ spec:
       lagThreshold: "50"
     authenticationRef:
       name: keda-trigger-auth-kafka-credential
-```
\ No newline at end of file
+```
```

Comment:
not clear what's the difference here may be git is going crazy :-)

Reply:
Think it's a diff thing for sure!